trident-use/worker-node-prep.adoc

```diff
@@ -57,7 +57,7 @@ NOTE: You should ensure that the NFS service is started up during boot time.
 Consider the following when using iSCSI volumes:
 
 * Each node in the Kubernetes cluster must have a unique IQN. *This is a necessary prerequisite*.
-* If using RHCOS version 4.5 or later, or RHEL or CentOS version 8.2 or later with the `solidfire-san` driver, ensure that the CHAP authentication algorithm is set to MD5 in `/etc/iscsi/iscsid.conf`.
+* If using RHCOS version 4.5 or later, or other RHEL-compatible Linux distribution, with the `solidfire-san` driver, ensure that the CHAP authentication algorithm is set to MD5 in `/etc/iscsi/iscsid.conf`.
```

Comment:
I think this may have been been fixed in Element Software 12.5 (see Jira ELEM-13770)

Maybe take a look at that issue and if it's been closed, you can make a change in this segment to say "with Element Software 12.3 or earlier, and the `solidfire-san` driver"

Existing KB:

https://kb.netapp.com/Advice_and_Troubleshooting/Flash_Storage/SF_Series/Newer_versions_of_Linux_open-iSCSI_initiator_for_environments_like_OpenShift_might_not_work

Reply:
HI @scaleoutsean, are you referring to the new language or the old reference to CentOS 8.2+? We removed the CentOS reference and replaced it with "or other RHEL-compatible Linux distribution". 

Adding @gnarl for awareness and input.

Reply:
Hi @scaleoutsean, very sorry for the delay on this!

Yes, CentOS 8 references have been removed. Per @gnarl, the MD5 algorithm is in [Prepare the worker node](https://docs.netapp.com/us-en/trident/trident-use/worker-node-prep.html#node-service-discovery).